### DISCUSSION TITLE
Project deployed to vercel, can't connect to mongodb

### DISCUSSION THREAD

**Discussion by rashikasahu35** [vercel/community#5708] [Help]
### Question

I have deployed express server on vercel, and integrated vercel and MongoDB using [this link](https://www.mongodb.com/docs/atlas/reference/partner-integrations/vercel/), 

have updated the MONGODB_URI in env var, then also getting this error
<img width="931" alt="image" src="https://github.com/vercel/community/assets/93260606/e4e4e5b9-b65f-4b37-b769-ada05477acf2">

these are my logs   
<img width="612" alt="image" src="https://github.com/vercel/community/assets/93260606/6fb1dcce-beac-46a7-af4e-944673738388">. 

this is my index.js file 
<img width="455" alt="image" src="https://github.com/vercel/community/assets/93260606/9e2ea9bb-1730-4f17-a52a-8374b650745c">

Please do reply.

**Comment by aldosch** [comment]
Hi @logic-found,

Thank you for sharing your findings. We really appreciate your patience, I understand this was raised a long time ago.

[There is a related discussion here](https://github.com/vercel/vercel/discussions/11178#discussioncomment-9503785). We encourage you to follow [**NODE-6179**](https://jira.mongodb.org/browse/NODE-6179) which is a bug report submitted to MongoDB covering the same connection timeout problem. If you could share any additional context or troubleshooting attempts there it would be very helpful.

Thanks,
Aldo
